packages/manager/modules/pci/src/projects/project/kubernetes/add/add.controller.js

```diff
@@ -49,6 +53,8 @@ export default class {
     };
 
     this.loadMessages();
+
+    this.$anchorScroll('addMessages');
```

Comment:
id already exists in the DOM ?

Reply:
mistake :), initially for a test.
removed